### DISCUSSION TITLE
0.2.7-bate

### DISCUSSION THREAD

**Discussion by tomakino** [tomakino/CSLyric#15] [Announcements]
1. 修复已知bug
1. 增强自定义歌词颜色功能
1. 支持LX音乐

<hr /><em>This discussion was created from the release <a href='https://github.com/tomakino/CSLyric/releases/tag/0.2.7-bate'>0.2.7-bate</a>.</em>

**Comment by lurve1314** [comment]
好像不是bate，而是beta😝
